Unithereum.CodeGen/Config.cs

```diff
@@ -0,0 +1,279 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Unithereum.CodeGen
+{
+    public class Config
+    {
+        private static readonly HashSet<string> CsharpKeywords = new HashSet<string>
+        {
+            "abstract", "as", "base", "bool", "break", "byte", "case", "catch", "char", "checked", "class", "const",
+            "continue", "decimal", "default", "delegate", "do", "double", "else", "enum", "event", "explicit", "extern",
+            "false", "finally", "fixed", "float", "for", "foreach", "goto", "if", "implicit", "in", "int", "interface",
+            "internal", "is", "lock", "long", "namespace", "new", "null", "object", "operator", "out", "override",
+            "params", "private", "protected", "public", "readonly", "ref", "return", "sbyte", "sealed", "short",
+            "sizeof", "stackalloc", "static", "string", "struct", "switch", "this", "throw", "true", "try", "typeof",
+            "uint", "ulong", "unchecked", "unsafe", "ushort", "using", "virtual", "void", "volatile", "while"
+        };
+
+        /// <summary>
+        ///     Absolute path to `dotnet` executable.
+        /// </summary>
+        public string? DotnetPath { get; }
+
+        /// <summary>
+        ///     Namespace prefix for generated contract service classes.
+        ///     <para>Default: (Application.productName).ContractServices</para>
+        /// </summary>
+        public string NsPrefix { get; }
```

Comment:
I've set it non-nullable because it would fallback to default or throw exception if property is not given or invalid.
I think rather setting `DotnetPath` also non-nullable might be better to make consistent, throwing early in Config constructor if given path is invalid.

And I agree `namespacePrefix` would be better. I'll make changes for it.